dockers/docker-orchagent/swssconfig.sh

```diff
@@ -29,6 +29,26 @@ function fast_reboot {
   esac
 }
 
+function config_reload {
+    if [[ -f /config-reload-restore ]];
+    then
+      if [[ -f /fdb.json ]];
+      then
+        swssconfig /fdb.json
+        mv -f /fdb.json /fdb.json.1
+      fi
+
+      if [[ -f /arp.json ]];
+      then
+        swssconfig /arp.json
```

Comment:
I think, may be we should not restore the `arp.json` entries to APP_DB since we are directly adding to kernel. If we add to APP_DB, arp_refresh script kick in and can result in failed entries temporarily.

Reply:
Ack, will fix